src/app/components/AddButton/AddButton.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import classes from './AddButton.module.css';
 import { GiLightBackpack } from 'react-icons/gi';
+import { Link } from 'react-router-dom';
 
 export default function AddButton(): JSX.Element {
   return (
     <div
```

Comment:
The div seems unneccessary. Have you tried to use the augmentad UI directly on the link?

Reply:
yep that is unfortunately afaik not working.